Source/Core/VideoCommon/VideoConfig.cpp

```diff
@@ -69,9 +69,12 @@ void VideoConfig::Load(const std::string& ini_file)
   settings->Get("OverlayStats", &bOverlayStats, false);
   settings->Get("OverlayProjStats", &bOverlayProjStats, false);
   settings->Get("DumpTextures", &bDumpTextures, false);
-  settings->Get("HiresTextures", &bHiresTextures, false);
-  settings->Get("ConvertHiresTextures", &bConvertHiresTextures, false);
-  settings->Get("CacheHiresTextures", &bCacheHiresTextures, false);
+  if (!settings->Get("CustomTextures", &bCustomTextures, false))
+  {
+	  settings->Get("HiresTextures", &bCustomTextures, false);
+  }
```

Comment:
While I'm trying to figure out how to do this .. What kind of technical reason is behind this requirement? I see a lot of tabs everwhere else.